pl_examples/hydra_examples/pl_template.py

```diff
@@ -20,21 +20,24 @@ def main(cfg: DictConfig):
     # ------------------------
     # 1 INIT LIGHTNING MODEL
     # ------------------------
-    model = LightningTemplateModel(cfg)
+    model = LightningTemplateModel(OmegaConf.masked_copy(cfg, ["data", "model", "scheduler", "opt"]))
     # ------------------------
     # 2 INIT TRAINER
     # ------------------------
-
-    callbacks = (
-        [hydra.utils.instantiate(c) for c in cfg.callbacks.callbacks_list] if "callbacks_list" in cfg.callbacks else []
-    )
+    callbacks = None
+    if cfg.callbacks:
+        callbacks = (
+            [hydra.utils.instantiate(c) for c in cfg.callbacks.callbacks_list]
+            if "callbacks_list" in cfg.callbacks
+            else []
+        )
 
     trainer = Trainer(
         **cfg.trainer,
-        logger=conf_init(cfg, "logger"),
-        profiler=conf_init(cfg, "profiler"),
-        checkpoint_callback=conf_init(cfg, "checkpoint"),
-        early_stop_callback=conf_init(cfg, "early_stopping"),
+        logger=hydra.utils.instantiate(getattr(cfg, "logger")),
```

Comment:
cfg.logger is not good?

Reply:
Also, lose the hydra.utils, just import instantiate directly.

```
from hydra.utils import instantiate
```

Reply:
Okay fixed the instantiate issue, what is the issue with cfg.logger?